OneCommunityGlobal/HighestGoodNetworkApp
# Anny create quick setup function and modal for new team members

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/98345990/634e463a-0720-4e7b-99d3-fe23eeff0406)

## Related PRS (if any):
This frontend PR is related to the[ #644  backend PR](https://github.com/OneCommunityGlobal/HGNRest/pull/644).

## Main changes explained:
- Created the QuickSetUpModal
- Only can be seen for Owner and Admin
- Created a new title with team and project to assign
- Setup the title with only a click and save the change

## How to test:
1. check into current branch at frontend and backend side
2. do `npm install`, `npm run build` and `npm run start:local`  to run this PR locally
3. Clear site data/cache
4. log as admin user or Owner to add a title and interact with quick-setup code
5. go to View profile→ Add a Title → Click on the Quick-Setup code
6. check if the userprofile updates


## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/98345990/4a5f27d1-eb20-41dc-9fc8-276105ce96c5
